File: problem-3/author-solutions/b.cpp
#include<iostream>
#include<vector>
#include<algorithm>
using namespace std;
int main(void) {
    int n; cin>>n;
    vector<int> a(n), b(n), v;
    for(int i=0; i<n; i++) cin>>a[i];
    for(int i=0; i<n; i++) cin>>b[i];
    for(int i=0; i<n; i++) {
      for(int j=0; j<n; j++) {
        if (a[i]==b[j]) {
          v.push_back(a[i]);
        }
      }  
    }
    sort(v.begin(), v.end());
    for(auto x: v) cout<<x<<' ';
    return 0;
}

File: problem-3/author-solutions/b.py
n = int(input())
a = [int(x) for x in input().split()]
b = [int(x) for x in input().split()]
v = []
for x in a:
  for y in b:
    if x==y:
      v.append(x)
v.sort()
for x in v:
  print(x, end=' ')